module/Decision/src/Decision/Mapper/Organ.php

```diff
@@ -121,6 +121,8 @@ public function findByAbbr($abbr, $type = null)
             $qb->andWhere('o.type = :type')
                 ->setParameter('type', $type);
         }
+        $qb->orderBy('o.foundationDate', 'DESC')
+            ->setMaxResults(1);
```

Comment:
I'm note sure about the side-effects here. The `findByAbbr` method might be used in places where we don't use this assumption.

At the very least you should document the assumption you're using here in the docblock. And you should check if it is not (indirectly) used anywhere that this might cause issues. If that is the case, fix those issues or add a parameter to make this behavior opt-in.

Reply:
@kokx `findByAbbr` is only used by `findOrganByAbbr`, which is again only used once (in `OrganController` for the `organAction`). So it should not be a problem.

However, I will add an additional parameter to the method to make it opt-in and explain its functionality in the docblock.